test/integration-tests/lib/mod.rs

```diff
@@ -250,7 +250,7 @@ pub fn start_sv2_translator(upstream: SocketAddr) -> (TranslatorSv2, SocketAddr)
 
     let min_extranonce2_size = 4;
 
-    let config = translator_sv2::proxy_config::ProxyConfig::new(
+    let config = translator_sv2::proxy_config::TranslatorProxyConfig::new(
```

Comment:
Can you please change the filename `translator/src/lib/proxy_config.rs` to `translator/src/lib/config.rs`?